x-pack/plugins/viz_editor/public/pseudo_plugins/xy_chart_plugin/xaxis_editor.tsx

```diff
@@ -23,17 +23,33 @@ export function XAxisEditor({
     const operation: SelectOperation =
       field.type === 'date'
         ? { operation: 'date_histogram', argument: { field: field.name, interval: '1m' } }
-        : { operation: 'column', argument: { field: field.name } };
+        : { operation: 'terms', argument: { field: field.name, size: 5 } };
```

Comment:
I've seen this in a few places, and I'm worried that this is not very reusable. There is a function I wrote for this in `common` called `fieldToOperation(field, operationName)` that fills out a full operation by template

Reply:
Makes sense, thanks.